### DISCUSSION TITLE
Is it possible to edit commands in your steam comment service bot?

### DISCUSSION THREAD

**Discussion by AKCH1234** [3urobeat/steam-comment-service-bot#229] [Q&A] [answered]
I've been wanting to change the text and make it look prettier, is that possible? 
![image](https://github.com/3urobeat/steam-comment-service-bot/assets/34775711/57fd4832-3a05-475a-9ec0-4b96bb198d3f)
 Let me know.

**Comment by 3urobeat** [comment, marked as answer]
Hey, yes, this is possible!  
Take a look at the [customlang wiki page](https://github.com/3urobeat/steam-comment-service-bot/blob/master/docs/wiki/customlang_doc.md) and let me know if anything is unclear ^-^

**Reply by AKCH1234** [reply]
What about editing commands, is that also possible?

**Reply by AKCH1234** [reply]
Commands that already exist^

**Reply by 3urobeat** [reply]
What exactly do you mean? You can edit all responses every command sends in the customlang file.  

You cannot modify how an existing command itself works (aka the code), but you can add new commands by [creating a plugin](https://github.com/3urobeat/steam-comment-service-bot/blob/master/docs/wiki/creating_plugins.md), if you know how to program.
